src/main/java/fr/openent/competences/controllers/SubTopicController.java

```diff
@@ -0,0 +1,46 @@
+package fr.openent.competences.controllers;
+
+import fr.openent.competences.Competences;
+import fr.openent.competences.constants.Field;
+import fr.openent.competences.security.AccessVisibilityAppreciation;
+import fr.openent.competences.service.SubTopicService;
+import fr.openent.competences.service.impl.DefaultSubTopicService;
+import fr.wseduc.rs.ApiDoc;
+import fr.wseduc.rs.Get;
+import fr.wseduc.rs.Post;
+import fr.wseduc.security.ActionType;
+import fr.wseduc.security.SecuredAction;
+import fr.wseduc.webutils.request.RequestUtils;
+import io.vertx.core.http.HttpServerRequest;
+import org.entcore.common.controller.ControllerHelper;
+import org.entcore.common.http.filter.ResourceFilter;
+
+import static fr.openent.competences.constants.Field.IDSTRUCTURE;
+import static org.entcore.common.http.response.DefaultResponseHandler.arrayResponseHandler;
+
+public class SubTopicController extends ControllerHelper {
+
+    SubTopicService subTopicService;
+    public SubTopicController() {
+        this.subTopicService = new DefaultSubTopicService(Competences.COMPETENCES_SCHEMA, Field.SERVICE_SUBTOPIC);
+    }
+
+    @Post("/subtopics/services/update")
+    @ApiDoc("set SubtopicsServices")
```

Comment:
Si tu as fait de nouvelles API, faudrait que tu la documentes et que tu mettes ton lien dans la description: 

_"I have made corresponding changes to the documentation (API Doc etc...) - (must specify in Description for target version)_"